test/scripts/e2e_subs/limit-swap-test.sh

```diff
@@ -42,7 +42,7 @@ echo "closeout part b, asset trader"
 # quick expiration, test closeout
 
 ROUND=$(goal node status | grep 'Last committed block:'|awk '{ print $4 }')
-SETUP_ROUND=$((${ROUND} + 10))
+SETUP_ROUND=$((${ROUND} + 20))
```

Comment:
It failed for me once since the transaction missed the "last valid" deadline by one round, so I extended it. I believe the reason is that for this test, the rounds are at least 2x faster with this change.